src/chapters/scope/introduction.rst

```diff
@@ -0,0 +1,144 @@
+Introduction
+============
+
+.. index:: ! scope
+
+Where variables are declared and initialized in the code can effect whether or not they can be used.
+**Scope** refers to the ability to access and use the variable.
+
+.. admonition:: Example 
+
+	.. sourcecode:: js
+
+		let a = 0;
+
+		function coolFunction() {
+			let b = 0;
+		}
+
+	``a`` is accessible *inside* and *outside* of ``coolFunction()``. 
+	``b`` is only accessible *inside* of ``coolFunction()``.
+
+Block/Local Scope
+-----------------
+
+.. index:: ! local scope
+
+**Local scope** refers to variables declared and initialized inside a function.
```

Comment:
"**Local scope** refers to variables declared and initialized inside a function or block."

Reply:
Thanks!